airlock/views/workspace.py

```diff
@@ -135,6 +135,7 @@ def workspace_view(request, workspace_name: str, path: str = ""):
             # for code button
             "code_url": code_url,
             "return_url": "",
+            "is_output_checker": request.user.output_checker or False,
```

Comment:
Wasn't sure if this was the right thing to do?

Reply:
Shouldn't be needed for workspace files, I think? If it's needed for the template validation, the `or False` isn't needed (it's a boolean)

Reply:
Lovely, I'll remove that.